src/opendr/perception/semantic_segmentation/dependencies.ini

```diff
@@ -0,0 +1,9 @@
+[runtime]
+# 'python' key expects a value using the Python requirements file format
+#  https://pip.pypa.io/en/stable/reference/pip_install/#requirements-file-format 
+python=torch==1.5.1
+       torchvision==0.6.1
+       tqdm==4.47.0
+       imgaug==0.4.0
+       PIL==7.0.0
```

Comment:
Already merged tools are using:
* `torch` package version 1.7.1
* `torchvision` package version 0.8.2
* `tqdm` package version 4.54.0

Could you please check if these versions is working for you as well and update it here?
We should use the same package version for all the methods in the toolkit.